etl/steps/data/grapher/democracy/2023-03-02/vdem.meta.yml

```diff
@@ -0,0 +1,164 @@
+        description_from_producer: |-
+          The index (<code>v2x_polyarchy</code>) is formed by taking the average of, on the one hand, the weighted average of the indices measuring freedom of association thick (<code>v2x_frassoc_thick</code>), clean elections (<code>v2xel_frefair</code>), freedom of expression (<code>v2x_freexp_altinf</code>), elected officials (<code>v2x_elecoff), and suffrage (<code>v2x_suffr</code>) and, on the other, the five-way multiplicative interaction between those indices. This is half way between a straight average and strict multiplication, meaning the average of the two. It is thus a compromise between the two most well known aggregation formulas in the literature, both allowing partial "compensation" in one sub-component for lack of polyarchy in the others, but also punishing countries not strong in one sub-component according to the "weakest link" argument. The aggregation is done at the level of Dahl’s subcomponents with the one exception of the non-electoral component. The index is aggregated using this formula:
+
+          <code>v2x_polyarchy = .5 ∗ MPI + .5 ∗ API</code>
+
+          <code>= .5 ∗ (v2x_elecoff ∗ v2xel_frefair ∗ v2x_frassoc_thick∗ v2x_suffr ∗ v2x_freexp_altinf) + .5 ∗ ((1/8) ∗ v2x_elecoff + (1/4) ∗ v2xel_frefair + (1/4) ∗ v2x_frassoc_thick + (1/8) ∗ v2x_suffr + (1/4) ∗ v2x_freexp_altinf)</code>
+
+          \[Text from [V-Dem Codebook v13](https://www.v-dem.net/documents/24/codebook_v13.pdf)\]
+        description_processing: |-
+          We ingested the original data from the [V-Dem project website](https://v-dem.net/data/the-v-dem-dataset/) and processed it with Stata.
+
+          We standardized the names of countries and regions by using the _[Our World in Data](https://ourworldindata.org/world-region-map-definitions)_ [standard entity names](https://ourworldindata.org/world-region-map-definitions).
+
+          We expanded the years and countries covered by V-Dem. To include more of the period when current countries were still non-sovereign territories, we identified the historical entity they were a part of and used that regime’s data whenever available.
+
+          For example, V-Dem only provides regime data since Bangladesh’s independence in 1971. There is, however, regime data for Pakistan and the colony of India, both of which the current territory of Bangladesh was a part. We, therefore, use the regime data of Pakistan for Bangladesh from 1947 to 1970, and the regime data of India from 1789 to 1946. We did so for all countries with a past or current population of more than one million.
+
+          We calculated regional and global averages of the Electoral Democracy Index and its sub-indices, weighted and unweighted by population.
+
+          All code and data is available [on GitHub](https://github.com/owid/notebooks/tree/main/BastianHerre/democracy).
+        presentation:
+          title_variant: V-Dem
+          attribution_short: V-Dem
+          topic_tags:
```

Comment:
I would remove this to avoid repetition in the text `V-Dem - V-Dem`

Reply:
Good catch, thanks for fixing.

Reply:
Hmm, it shows this after removing it

`How does the producer of this data - undefined - describe this data?`

I'm going to leave it there until we have sensible fallbacks.